lib/foreman_tasks/dynflow/daemon.rb

```diff
@@ -68,5 +87,43 @@ def run_background(command = 'start', options = {})
     def world
       ForemanTasks.dynflow.world
     end
+
+    private
+
+    def daemons_options(command, options)
+      {
+	:multiple => true,
+	:dir => options[:pid_dir],
+	:log_dir => options[:log_dir],
+	:dir_mode => :normal,
+	:monitor => true,
+	:log_output => true,
+	:log_output_syslog => true,
+        :monitor_interval => options[:memory_polling_interval] / 2,
```

Comment:
This setting is responsible for checking if the daemon is still alive, and spawning a new one in case it's not. If the process is killed by memory polling, I would like to see it spawns pretty quickly (in this implementation it would be at most half of the polling interval).

I am changing the logic to be at least as frequent as the default value (which is 30 sec).